inst/include/kernels/Kernel.hpp

```diff
@@ -140,15 +140,15 @@ namespace exageostat::kernels {
          * @return The value of P.
          *
          */
-        [[nodiscard]] int GetPValue() const;
+        [[nodiscard]] int GetP() const;
```

Comment:
P is the number of data variables. I think it is better to rename P to NumVars

Reply:
In the C there's P and number of parameters. Same In C++ so making P NumVars I think will be similar to num_params

else if (strcmp(arguments.kernel_fun, "univariate_spacetime_matern_stationary") == 0) {
        num_params = 7;
        p = 1;
    } else if (strcmp(arguments.kernel_fun, "bivariate_spacetime_matern_stationary") == 0) {
        num_params = 10;
        p = 2;

Reply:
params and vars are two different things. `Variables` here is the data variables, `parameters` is the model parameters. no we can use `num_params` and `num_vars`